### ISSUE TITLE
Save Start Crash After updating to 0.3.3

### ISSUE THREAD

**Issue by SGZ-TW-TY-20** [raiguard/Factorio-LtnManager#111]
#### Describe the Bug

I updated the mod that had fixes
https://cdn.discordapp.com/attachments/697564089907675217/757893207773282334/unknown.png

#### To Reproduce

Steps to reproduce the behavior: Have save that used V 0.3.2 update to 0.3.3

1. Go to:
Restart game after update
2. Click on:
Choose the save u play on
3. (Scroll down to ?)
Wait a few sec the crash should appear
That error pops up.

https://cdn.discordapp.com/attachments/697564089907675217/757892073214050304/unknown.png

Be certain to reproduce the issue on the same save file that you upload:

[1.0 Bob-s Darkstar 3.zip](https://github.com/raiguard/Factorio-LtnManager/files/5260337/1.0.Bob-s.Darkstar.3.zip)

Please also provide your in-game username so I can reproduce the conditions exactly:
Nickname S_gamez, sgamez or Sgamez

#### Log file
[factorio-current.log](https://github.com/raiguard/Factorio-LtnManager/files/5260362/factorio-current.log)
[factorio-previous.log](https://github.com/raiguard/Factorio-LtnManager/files/5260363/factorio-previous.log)

Attach `factorio-current.log`, found in Factorio's [user data directory](https://wiki.factorio.com/Application_directory#User_data_directory).

**Comment by dwd89** [comment]
Error while running event LtnManager::on_tick (ID 0)
__LtnManager__/scripts/ltn-data.lua:455: attempt to concatenate field 'to' (a nil value)
stack traceback:
	__LtnManager__/scripts/ltn-data.lua:455: in function '?'
	__LtnManager__/scripts/ltn-data.lua:544: in function 'iterate'
	__LtnManager__/control.lua:157: in function <__LtnManager__/control.lua:153

**Comment by raiguard** [comment]
Thanks for the report, this has been fixed for the next release.

@dwd89 Please make a separate issue and upload your save file, that crash has nothing to do with this one.

**Comment by SGZ-TW-TY-20** [comment]
am waiting for the fix before I play on my game :/

**Comment by raiguard** [comment]
I was hoping to be able to wait for @dwd89, but I can release it now if you're waiting on it.

**Comment by SGZ-TW-TY-20** [comment]
Thnaks

**Comment by SGZ-TW-TY-20** [comment]
https://cdn.discordapp.com/attachments/697564089907675217/758033737329016966/unknown.png

The issue accrued again totally identical to #111
https://cdn.discordapp.com/attachments/697564089907675217/758036185065980044/unknown.png

Just now with new version & now with running trains like similar to   

https://cdn.discordapp.com/attachments/697564089907675217/757725073779064832/unknown.png

Have the same issue. just mine are not in Russian

_Originally posted by @SGZ-TW-TY-20 in https://github.com/raiguard/Factorio-LtnManager/issues/110#issuecomment-696413833_

**Comment by raiguard** [comment]
Thanks, see #112. This ended up being quite the gnarly bug!

@dwd89 I'm fairly confident that that solved the issue you were having as well, please let me know as soon as you can!

**Comment by SGZ-TW-TY-20** [comment]
Thanks. i can defiantly confirm it was annoying.

**Comment by dwd89** [comment]
Everything works on Razie, I was playing on the previous version waiting for a solution.
